server/admin_management/api/app/main.py

```diff
@@ -0,0 +1,68 @@
+import logging.config
+import os.path
+
+from fastapi import APIRouter, FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from starlette.responses import RedirectResponse
+from mangum import Mangum
+
+from api.config.config import get_root_path, get_allow_origins
+from api.app.routers import router_smoke_test
+
+
+logConfigFile = os.path.join(
+    os.path.dirname(__file__), "..", "config", "logging.config"
+)
+logging.config.fileConfig(logConfigFile, disable_existing_loggers=False)
+LOGGER = logging.getLogger("api.app.main")
+
+
+apiPrefix = ""
+description = """
+Forest Access Management API used by the Forest Access Management application
+to define admin access to forest applications.
+"""
+
+
+def custom_generate_unique_id(route: APIRouter):
+    # The outcome of this is on openapi spec's "operationId" field.
+    # When api client is generated, this will be used for function/method name.
+    return f"{route.name}"  # f"{route.tags[0]}-{route.name}"
+
+
+app = FastAPI(
+    title="Forest Access Management - FAM - Admin Management API",
+    description=description,
+    version="0.0.1",
+    contact={
+        "name": "Guy Lafleur",
+        "url": "https://en.wikipedia.org/wiki/Guy_Lafleur",
+        "email": "guy.lafleur@montreal.canadians.ca",
+    },
+    license_info={
+        "name": "Apache 2.0",
+        "url": "https://www.apache.org/licenses/LICENSE-2.0.html",
+    },
+    root_path=get_root_path(),
+    generate_unique_id_function=custom_generate_unique_id,
+)
+
```

Comment:
oh yeah, I remember that. The reason I didn't add it now is because define a specific OpenAPI version is for the frontend code generator, but we're not doing it yet. For this ticket I only added really basic stuff and what is needed for now. We'll have another ticket to think about the code generator anyway, maybe we can add by that time?

Reply:
Yes, I was wondering maybe you were leaving it for some reason.
Sure, we can add that later, just this kind of workaround (nobody prefer) can be tricky if we forgot.